airbyte-webapp/src/components/LoadingBackdrop/LoadingBackdrop.module.scss

```diff
@@ -0,0 +1,28 @@
+@use "../../scss/colors";
+
+.cover {
```

Comment:
Stylelint will flag this, should be `%cover` and `@extend %cover`

Reply:
Oh, okay, will fix it